examples/logger.py

```diff
@@ -85,11 +148,21 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame = ConfirmClearingFrame()
         self.confirmFrame.confirmed.connect(self.clearLog)
+        self.loggerFrame.levelButton.clicked.connect(self.selectLevel)
+        self.setlevelFrame = SetLevelFrame()
+        # define handler
+        self.handler = LoggingHandler(self.addLog)
+        # arbitrary format
+        fs ="%(name)s %(message)s"
+        formatter = logging.Formatter(fs)
+        self.handler.setFormatter(formatter)
+        logger.addHandler(self.handler)
```

Comment:
Note that you only need the root logger object here.
Therefore, you can just write:
```python
logging.getLogger().addHandler(self.handler)
```